### PR TITLE
[APB-1204][ph] fixed tests failure due to config bug

### PR DESCRIPTION
**Pull request by PaulHodgson**
Please remember to run 'acc:test' when doing the PR as I forgot this yesterday...

### CODE REVIEW

File: test/uk/gov/hmrc/agentclientauthorisation/controllers/AgencyControllerSpec.scala [hunk 1]
```diff
@@ -185,12 +182,11 @@ class AgencyControllerSpec extends ACAFUnitSpec with MockitoSugar with BeforeAnd
       agentAuthStub(agentAffinityAndEnrolments)
       when(mockInvitationsConnector.createInvitation(any(), any())(any())).thenReturn(Future successful Some("location"))
       when(mockInvitationsConnector.getInvitation(eqs("location"))(any[HeaderCarrier]())).thenReturn(Future successful Some(testInvite))
-      when(mockAppConfig.clientInvitationUrl).thenReturn("clientInvitationUrl/")
 
       val response = await(agencyController.inviteClient(FakeRequest().withFormUrlEncodedBody(validForm: _*)))
 
       status(response) shouldBe OK
-      contentAsString(response) should include(s"clientInvitationUrl/$nino")
+      contentAsString(response) should include(s"agent-client-authorisation/clients/invitations/$nino")
```
Comment (frankoid) [inline test/uk/gov/hmrc/agentclientauthorisation/controllers/AgencyControllerSpec.scala]: this doesn't feel as much like a unit test as it did before because it's now using the real FrontendAppConfig which is presumably loading the play config

do you know why the change made in this PR to `AgencyController` was needed?

Comment (frankoid) [reply test/uk/gov/hmrc/agentclientauthorisation/controllers/AgencyControllerSpec.scala]: perhaps a better fix might be to add a binding to the GuiceModule:
```scala
    bind(classOf[AppConfig]).toInstance(FrontendAppConfig)
```

(not tested)

Comment (PaulHodgson) [reply test/uk/gov/hmrc/agentclientauthorisation/controllers/AgencyControllerSpec.scala]: I'd originally added a dependency on AppConfig in the controller and had thought to mock this out in the test. This caused the failure as the dependency injection broke - I didn't run 'acc:test' locally.

When I revisited this I reasoned that as AuthorisedAgent handleFailure accesses the property directly I'd do the same here as its a simpler solution.

The assertion in the test does rely on the property in doesn't rely on the config-specific portion of the url. Granted it spans more than 1 unit. I considered just asserting the presence of the nino but that seemed less clear and less useful.

Comment (PaulHodgson) [reply test/uk/gov/hmrc/agentclientauthorisation/controllers/AgencyControllerSpec.scala]: Perhaps re-factor the code so that the controller is the home of this property?

  azy val clientInvitationUrl = s"$agentClientAuthorisationFrontendBaseUrl/agent-client-authorisation/clients/invitations/"
